Wildfire verdicts c2 fix (#24493)
demisto/content: Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/2_1_15.md

#### Integrations
##### Palo Alto Networks WildFire v2

> - Fixed an issue with wildfire-get-verdict command where c2 results were returned as the integer 5 instead of c2.

Reviewer: ```suggestion
- Fixed an issue where the ***wildfire-get-verdict*** command has returned the integer 5 as a result of the c2 verdict.
```
